server/server.go

```diff
@@ -1475,6 +1506,14 @@ func (s *Server) reloadConfigFromKV() error {
 	return nil
 }
 
+func (s *Server) loadRateLimitConfig() {
```

Comment:
Why do we need it?

Reply:
The configuration of the Limiter differs from that of EnableRateLimit. The value of EnableRateLimit is fetched from PersistOption on every request, but Limiter does not. So after resigning leader, we should reload the associated configuration.